app/components/Views/confirmations/components/TransactionReview/index.js

```diff
@@ -324,9 +321,11 @@ class TransactionReview extends PureComponent {
       fiatValue,
       approveTransaction,
     });
-    InteractionManager.runAfterInteractions(() => {
-      AnalyticsV2.trackEvent(MetaMetricsEvents.TRANSACTIONS_CONFIRM_STARTED);
-    });
+
+    this.props.metrics.trackEvent(
```

Comment:
Destructure metrics on L294
```suggestion
    metrics.trackEvent(
```